frontend/public/locales/fr/learn/main-sources-of-retirement-income.json

```diff
@@ -2,205 +2,323 @@
   "breadcrumbs": {
     "learn": "Apprendre"
   },
-  "meta": {
```

Comment:
`meta.description` key shouldn't be removed and it's already translated.

Reply:
it's actually further down the json, but the json is alphabetically sorted. It wasn't removed, just ... moved.